.travis.yml

```diff
@@ -1,21 +1,23 @@
 sudo: required
-dist: trusty
+dist: jammy
 language: node_js
 node_js:
-  - '12'
+  - '14.20.1'
```

Comment:
Same comment as in systelab-translate: shouldn't we move the version 16 as this is the latest LTS?
According to the docs, the 16.13.0 is preinstalled on the "jammy" image: https://docs.travis-ci.com/user/reference/jammy/#javascript-and-nodejs-support